README.md

```diff
@@ -1,2 +1,145 @@
-# Leaderboard-setup-project
-Leaderboard: setup project
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:movie_camera: Project Presentation](#project-presentation)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+
+**[Leaderboard ]** "Leaderboard" is a JavaScript project focused on creating a list app using webpack and ES6 features, particularly modules. The initial version aims to implement core functionality based on a wireframe, prioritizing functionality over styling. Subsequent activities will involve integrating the Leaderboard API using JavaScript async and await, and enhancing the app with styling elements.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS and Javascript vanilla.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Plain JS with objects]**
+- **[JavaScript classes]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Presntation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+Walking through portfolio outline.
+
+- [Project Presentation Link] [#](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](Not available for this time)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+Use Git or checkout with SVN using the web URL.
+https://github.com/
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
```

Comment:
- [OPTIONAL] Good job for removing the Live demo link as suggested 🥇 I noticed that subheadings for **_Getting Started_** section is missing ⚠ Could you please include them and update its content according to the project you are working on?  Here's is an [example README](https://github.com/KanzaTahreem/Cryptonex) for you 💪
- [OPTIONAL] Please make sue to use the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) for your upcoming milestones. Since it was missed by the previous reviewer, that's why I'm making it as optional, however, it is highly recommended to follow the correct flow

Reply:
@KanzaTahreem, thank you very much for your review. However, I would like to highlight that I am using Gitflow as it is intended to be used. If you take a closer look, you'll notice that I am working on another branch, as expected in the Gitflow workflow.